service/history/historyBuilder.go

```diff
@@ -95,8 +98,8 @@ func (b *historyBuilder) AddWorkflowTaskScheduledEvent(taskQueue *taskqueuepb.Ta
 }
 
 func (b *historyBuilder) AddTransientWorkflowTaskScheduledEvent(taskQueue *taskqueuepb.TaskQueue,
-	startToCloseTimeoutSeconds int32, attempt int64, timestamp int64) *historypb.HistoryEvent {
-	event := b.newTransientWorkflowTaskScheduledEvent(taskQueue, startToCloseTimeoutSeconds, attempt, timestamp)
+	startToCloseTimeoutSeconds int32, attempt int64, time time.Time) *historypb.HistoryEvent {
```

Comment:
Did few conversions on `historyBuilder` but there are still many left. So we are in a mixed mode now: some func uses `time.Time` and some `timestamp int`.